recipe/meta.yaml

```diff
@@ -1,37 +1,119 @@
-{% set version = "5.5.2" %}
+{% set version = "5.7.0" %}
```

Comment:
maybe you should include `dev` in version?.. to be able to distinguish this and real `5.7.0` releases

Reply:
Including `dev` in the version will complicate the specifications because `version` is used in file paths that do not include the `dev` part. I think it is not worth it as `dev` will be gone in a month or so anyway.

Reply:
But then again, how would you tell apart `5.7.0` built from `5.7.0-dev` tag and `5.7.0` built from `5.7.0` release?..

Reply:
I'll ask upstream if the next release could be `5.7.1` but we can use `build` number to distinguish between 5.7.0-dev and 5.7.0.

Reply:
Confirmed: the next release will be 5.7.0.

Let me try to introduce `base_version=5.7.0` and have `version=5.7.0dev-20210429` as tagged by the upstream..